webpack.config.js

```diff
@@ -53,11 +53,11 @@ const nodeConfig = {
     ...baseConfig.resolve,
     alias: {
       platform: path.resolve(nodePath, 'platform'),
-      'platform-http': path.resolve(nodePath, 'lib', 'util', 'http'),
-      'platform-bufferutils': path.resolve(nodePath, 'lib', 'util', 'bufferutils'),
-      'platform-defaults': path.resolve(nodePath, 'lib', 'util', 'defaults'),
-      'platform-crypto': path.resolve(nodePath, 'lib', 'util', 'crypto'),
-      'platform-transports': path.resolve(nodePath, 'lib', 'transport'),
+      'platform-http': path.resolve(nodePath,  'lib', 'util', 'http'),
```

Comment:
You added some extraneous whitespace here and on line 161

Reply:
Happy for you to add webpack.config.js to our prettier targets as part of this PR